Tests/SwiftCliCoreTests/ChartConstructorTest.swift

```diff
@@ -105,6 +105,21 @@ class RightDecreaseMiddlePrinterTests: XCTestCase {
 
 }
 
+class RightIncreaseMiddlePrinterTests: XCTestCase {
+    func testOneRightIncrease() throws {
+        let result = ChartConstructor().makeMiddleRowStitchCountChange(width: 5, rDiff: 1, lDiff: 0)
+        let expectedOutput = "├─┼─┼─┼─┼─┘\n"
+        expect(result).to(equal(expectedOutput))
+
+    }
+
+    func testOneMultiIncrease() throws {
```

Comment:
Do you mean `testMultipleRightIncrease`?

Reply:
Yes!